chainer_setup_build.py

```diff
@@ -388,6 +421,10 @@ def cythonize(extensions, force=False, annotate=False, compiler_directives={}):
 
     cython_cmdbase.extend(['--fast-fail', '--verbose', '--cplus'])
     for ext in extensions:
+        if all(not _is_changed(f) for f in ext.sources) and only_changed:
```

Comment:
I think that the calculation of md5 is not necessary for most users.
So, it is better that extra file is not made.
If you want to check the md5 of all files, following code is good.
`if all([not _is_changed(f) for f in ext.sources]) and only_changed:`